patches/minecraft/net/minecraft/network/handshake/ServerHandshakeNetHandler.java.patch

```diff
@@ -1,10 +1,11 @@
 --- a/net/minecraft/network/handshake/ServerHandshakeNetHandler.java
 +++ b/net/minecraft/network/handshake/ServerHandshakeNetHandler.java
-@@ -23,6 +23,7 @@
+@@ -23,6 +23,8 @@
     }
  
     public void func_147383_a(CHandshakePacket p_147383_1_) {
 +      if (!net.minecraftforge.fml.server.ServerLifecycleHooks.handleServerLogin(p_147383_1_, this.field_147386_b)) return;
++      net.minecraftforge.network.VanillaConnectionNetworkFilter.injectIfNecessary(this.field_147386_b);
```

Comment:
Can't this line be added to the method above? Why add it here and create a patch?

Reply:
I was under the impression you want to keep FML somewhat out of the "major game patches", so it seemed to me that FML calling into Forge code would be a bit strange. But I'll change it if that's no problem.